templates/zerver/emails/email_base_default.source.html

```diff
@@ -14,7 +14,21 @@
                         <a href="#" class="illustration">
                             {% block illustration %}{% endblock %}
                         </a>
-                        {% block preheader %}{% endblock %}
+                        <span style="display:none !important;
```

Comment:
Don't add the CSS here. Add the CSS in email.css file in the same folder. Then run `./tools/inline-email-css` and do the testing. We are in the process of removing the inline css from the remaining source files.